src/control/drpc/drpc_client.go

```diff
@@ -58,6 +58,7 @@ type ClientConnection struct {
 	socketPath string             // Filesystem location of dRPC socket
 	dialer     domainSocketDialer // Interface to connect to the socket
 	conn       domainSocketConn   // UDS connection
+	sequence   int64              // Increment each time we send
```

Comment:
what is the purpose of this sequence?

Reply:
This is the sequence number for matching the request to the response. We want to ensure it's unique to each dRPC request, at least within a given session context. We do something similar in the dRPC C code.